{"name": "test_get_error", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "browser = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"733c900d54d87595b36f2c8b3200e24c\")>\n\n    @allure.feature('Fail test')\n    @allure.story('Existence')\n    def test_get_error(browser):\n>       assert False\nE       assert False\n\ntests\\test_first_scenario.py:10: AssertionError"}, "attachments": [{"name": "Screenshot on failure", "source": "bbf06c74-5850-48b3-8a4f-b0a2774b1def-attachment.png", "type": "image/png"}], "start": 1750602345710, "stop": 1750602345710, "uuid": "7e80e3b1-879b-47a6-9bab-abefdf2cc605", "historyId": "22dfb66f240faba869d9b3567e973a6b", "testCaseId": "22dfb66f240faba869d9b3567e973a6b", "fullName": "tests.test_first_scenario#test_get_error", "labels": [{"name": "story", "value": "Existence"}, {"name": "feature", "value": "Fail test"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_first_scenario"}, {"name": "host", "value": "DESKTOP-KHTU12F"}, {"name": "thread", "value": "9632-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_first_scenario"}]}